src/Fantomas.Core/CodePrinter.fs

```diff
@@ -3045,30 +3045,26 @@ let genType (t: Type) =
             +> genSingleTextNode node.Closing
 
         let longExpression =
-            let genFields = col sepNln node.Fields genAnonRecordFieldType
-
-            let genMultilineAnonRecordTypeAlignBrackets =
-                let genRecord =
-                    sepOpenAnonRecdFixed
-                    +> indentSepNlnUnindent (atCurrentColumnIndent genFields)
+            let genAnonRecordFields = col sepNln node.Fields genAnonRecordFieldType
+
+            handleBracketStyle (function
+                | Aligned
+                | ExperimentalStroustrup ->
+                    genStruct
+                    +> sepOpenAnonRecdFixed
+                    +> indentSepNlnUnindent (atCurrentColumnIndent genAnonRecordFields)
                     +> sepNln
                     +> genSingleTextNode node.Closing
-
-                genStruct +> genRecord
-
-            let genMultilineAnonRecordType =
-                let genRecord =
-                    genOpening
-                    +> atCurrentColumn genFields
+                | Cramped ->
+                    genStruct
+                    +> genOpening
+                    +> atCurrentColumn genAnonRecordFields
                     +> addSpaceIfSpaceAroundDelimiter
-                    +> genSingleTextNode node.Closing
-
-                genStruct +> genRecord
-
-            ifAlignOrStroustrupBrackets genMultilineAnonRecordTypeAlignBrackets genMultilineAnonRecordType
+                    +> genSingleTextNode node.Closing)
 
         fun (ctx: Context) ->
             let size = getRecordSize ctx node.Fields
+
```

Comment:
Please also don't introduce these additional newlines.
Fantomas will combine two single lines of code without an additional new line.
That is the heuristic and we wish to respect that.

By adding the extra newline you took a stylistic decision that the previous developer on the same team didn't make. We use a code formatter to remove every personal touch any developer might make, so please keep an eye out for that.

Reply:
I'm not 100% sure if it's the case here but I noticed that when I ran the `FormatAll` build command it added a bunch of newlines for me. I thought I removed them but I guess not. Not sure why it would have done that